juniper_official/interface/check-optical-signal-loss-fec-tx-rx-power.rule

```diff
@@ -0,0 +1,420 @@
+                }
+            }
+            trigger interface-tx-laser-disabled-alarm {
+                frequency 1.5offset;
+                term is-tx-laser-disabled-alarm-off {
+                    when {
+                        matches-with "$tx-laser-disabled-alarm" false {
+                            ignore-case;
+                        }
+                    }
+                    then {
+                        status {
+                            color green;
+                            message "Interface $interface-name, Lane $lane-index Tx laser disabled alarm is OFF";
+                        }
+                    }
+                }
+                term is-tx-laser-disabled-alarm-exists {
+                    when {
+                        exists "$tx-laser-disabled-alarm";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Interface $interface-name, Lane $lane-index Tx laser disabled alarm is ON";
+                        }
+                    }
+                }
+            }
+            trigger interface-tx-loss-of-signal-functionality {
+                frequency 1.5offset;
+                term is-tx-loss-of-signal-functionality-off {
+                    when {
+                        matches-with "$tx-loss-of-signal-functionality-alarm" false {
+                            ignore-case;
+                        }
+                    }
+                    then {
+                        status {
+                            color green;
+                            message "Interface $interface-name, Lane $lane-index Tx loss of signal functionality alarm is OFF";
+                        }
+                    }
+                }
+                term is-tx-loss-of-signal-functionality-exists {
+                    when {
+                        exists "$tx-loss-of-signal-functionality-alarm";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Interface $interface-name, Lane $lane-index Tx loss of signal functionality alarm is ON";
+                        }
+                    }
+                }
+            }
+            trigger interface-optical-rx-high-power {
+                synopsis "high rx power usage";
+                description "Sets health based on the high rx power usage";
+                frequency 1.5offset;
+                term above_alarm_threshold {
+                    when {
+                        greater-than-or-equal-to "$optics-rx-power" "$rx-high-alarm-threshold";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Rx power $optics-rx-power is above alarm threshold $rx-high-alarm-threshold for interface $interface-name lane $lane-index.";
```

Comment:
Shorten message something like below

$interface-name, Lane $lane-index Rx power($optics-rx-power) is above alarm threshold($rx-high-alarm-threshold)